src/main/java/cdw/springtraining/gatekeeper/entites/Roles.java

```diff
@@ -0,0 +1,29 @@
+package cdw.springtraining.gatekeeper.entites;
+
+import jakarta.persistence.*;
```

Comment:
import only needed package and avoid importing all packages

Reply:
Removed wildcard